pawsitive/app/views/services/new.html.erb

```diff
@@ -0,0 +1,82 @@
+  </div>
+
+  <!-- Description Text Area -->
+  <div class="mb-4">
+    <%= f.label :description, "Description", class: "block text-sm font-medium text-gray-700" %>
+    <%= f.text_area :description, class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+  </div>
+
+  <!-- Availability Selection (Checkboxes) -->
+  <div class="mb-4">
+    <%= f.label :availability, "Availability", class: "block text-sm font-medium text-gray-700" %>
+    <% days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'] %>
+    <% days.each do |day| %>
+      <div class="flex items-center mb-2">
+        <%= label_tag "service[availability][#{day}][open]", "#{day} Open", class: "mr-2 text-sm" %>
+        <%= time_field_tag "service[availability][#{day}][open]", '08:00', class: "mr-4 mt-1 block w-1/4 rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+        <%= label_tag "service[availability][#{day}][close]", "Close", class: "mr-2 text-sm" %>
+        <%= time_field_tag "service[availability][#{day}][close]", '20:00', class: "mt-1 block w-1/4 rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+      </div>
+    <% end %>
+  </div>
+
+  <!-- price Selection (Text Inputs for Different Sizes) -->
+  <div class="mb-4">
+    <%= f.label :price, "Price", class: "block text-sm font-medium text-gray-700" %>
+    <div class="grid grid-cols-2 gap-4">
+      <div>
+        <%= label_tag "service[price][small_dog]", "Small Dog", class: "text-sm" %>
+        <%= number_field_tag "service[price][small_dog]", '', class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+      </div>
+      <div>
+        <%= label_tag "service[price][medium_dog]", "Medium Dog", class: "text-sm" %>
+        <%= number_field_tag "service[price][medium_dog]", '', class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+      </div>
+      <div>
+        <%= label_tag "service[price][large_dog]", "Large Dog", class: "text-sm" %>
+        <%= number_field_tag "service[price][large_dog]", '', class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+      </div>
+      <div>
+        <%= label_tag "service[price][giant_dog]", "Giant Dog", class: "text-sm" %>
+        <%= number_field_tag "service[price][giant_dog]", '', class: "mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-500 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" %>
+      </div>
+    </div>
+  </div>
+
+  <!-- Size Range Selection (Dropdowns) -->
```

Comment:
Hey Lincoln!
This code is well written, but I have some questions on the UX and business logic.
Why does the member choose between 2 size ranges for each size category?
As I understand this part, the member should just select which size categories (each representing a defined weight range) they accept.
This is really key for the filters, since the user just needs to select the size range(s) they want to search for. If there's a small size of 16-30 lbs and a medium of 16-40 lbs, which one will the user filter by? And if they can select both, isn't that very repetitive?

Reply:
my intention was to have a initial prompt in the input which would have a null value, however i got lots of errors so i added an extra value.